### PR TITLE
Redesign and update Dutch implementations page

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Add remaining sections (including new icons)
- [x] Add new header image
- [x] Update page description
- [x] Remove redundant partials introduced in previous commits
- [x] Remove unused icons (white)

### CODE REVIEW

File: source/localizable/implementaties/nl/_contact.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      <%= image_tag "team/marleen-renshoff.jpg", class: "avatar", alt: "Marleen Renshoff - Projectmanager" %>
+    </p>
+
+    <h3>
+      <%= t("implementaties.cta_heading") %>
+    </h3>
+    <p>
+      <%= t("implementaties.cta_text") %>
+    </p>
+  </div>
```
Comment (RonaldDijkstra) [inline source/localizable/implementaties/nl/_contact.html.erb:14]: This is a bit messy

Comment (michellusthof) [reply source/localizable/implementaties/nl/_contact.html.erb:14]: Rather use text instead of translatable strings?

Comment (RonaldDijkstra) [reply source/localizable/implementaties/nl/_contact.html.erb:14]: I meant the code itself.

**Comment by RonaldDijkstra** [comment]
Screenshot 2020-04-23 at 11 01 57

These texts are too wide

**Pull request merged**